plugin/trino-kinesis/src/test/java/io/trino/plugin/kinesis/TestRecordAccess.java

```diff
@@ -160,7 +182,8 @@ public void testStreamHasData()
     public void testJsonStream()
     {
         // Simple case: add a few specific items, query object and internal fields:
-        createJsonMessages(jsonStreamName, 4, 100);
+        createJsonMessages(jsonStreamName, 2, 100, false);
+        createJsonMessages(jsonStreamName, 2, 102, true);
```

Comment:
Let's make test parametric. You can use `io.trino.testing.DataProviders#trueFalse`.